src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    block-size: cover;
+    background-position: center;
+    background-image: url("../../src/images/star.svg");
+
+    list-style: none;
+  }
+
+  &--0 &__star:nth-child(-n+4) {
+    background-image: url("../../src/images/star-active.svg");
+  }
+}
```

Comment:
You were to re-use the solution from starts task
This won't work, as you have to support classes stars--0, stars--1, ..., stars--5 and display correct number of the stars respectively
Please see the task for "stars"

Reply:
You didn't fix this